pkg/resource/interfaces.go

```diff
@@ -28,13 +28,20 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/resource/unstructured/reference"
 )
 
-// A Conditioned may have conditions set or retrieved. Conditions are typically
+// A Conditioned may have conditions set or retrieved. Conditions typically
 // indicate the status of both a resource and its reconciliation process.
 type Conditioned interface {
 	SetConditions(c ...xpv1.Condition)
 	GetCondition(ct xpv1.ConditionType) xpv1.Condition
 }
 
+// A Statused may have status set or retrieved. Status is typically
+// a struct containing various runtime status properties.
+type Statused interface {
```

Comment:
I am wondering if we need this... perhaps we just need `Pave.SyncStatus(old, new)`?